### ISSUE TITLE
API access to list of icon names

### ISSUE THREAD

**Issue by opet** [nrkno/core-icons#23]
It would be nice to have access to a list of icon names (and categories?).

**Comment by eirikbacker** [comment]
@opet could you give an example on the format you wish to receive?
I started building a JSON like the one here (without .svg in each name):
```
{
  all: ['nrk-logo-nrk', 'nrk-logo-mp3', 'nrk-media-play'],
  logo: ['nrk-logo-nrk', 'nrk-logo-mp3'],
  media: ['nrk-media-play']
}
```
But any change is of course possible :)

**Comment by opet** [comment]
I guess that works pretty well. I don't really have a preference.

**Comment by eirikbacker** [comment]
Closed by #26
